packages/odyssey-react/src/components/Banner/Banner.module.scss

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.banner {
+  display: grid;
+  position: relative;
+  grid-template-areas: 'icon title content actions';
+  grid-template-columns: repeat(4, minmax(min-content, max-content));
+  grid-template-rows: 1fr;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding-block: $spacing-s;
+  padding-inline: $spacing-m;
+  background-color: cv('blue', '300');
+}
+
+.bannerIcon {
```

Comment:
I think we can safely remove `.banner` from the class names throughout. I did the same thing with Button initially, forgetting that the output class will be prefixed with the component.

Reply:
Done!